docs/README.md

```diff
@@ -0,0 +1,110 @@
+## Controllers
+### Clustermesh Controller
+#### Description
+This is the controller responsible for managing the mesh between the Clusters. It watches `clusters.cluster.x-k8s.io` and create the necessary cloud resources to establish the mesh between them leveraging on Crossplane CRs. 
```

Comment:
I think you need to explain what is a mesh (something like "network reachable clusters between each other in the cloud") (it may worth mentioning that today we only have support to AWS), even if you add techincally later what the controller does. Mesh/Clustermesh can mean a lot of things, including CNI/Service mesh level, which is not the case here

Reply:
Also, it is worth a diagram/drawing of a cluster mesh?

Reply:
I added a reference to the Glossary in the CR section